src/main/java/pl/javastart/task/ArrayUtils.java

```diff
@@ -0,0 +1,55 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+public class ArrayUtils {
+
+    private Scanner scanner = new Scanner(System.in);
+    private int size;
+    private double[] array;
+    private double sum = 0;
+
+    void getArraySize() {
+        System.out.println("Podaj rozmiar tablicy");
+        size = scanner.nextInt();
+    }
+
+    void createArray(int size) {
+        System.out.println("Wprowadź " + size + " liczb:");
+        array = new double[size];
+        for (int i = 0; i < array.length; i++) {
+            array[i] = scanner.nextDouble();
+        }
+    }
+
+    void sum(double[] array) {
+        for (double element : array) {
+            sum += element * element;
+        }
+        System.out.println("Suma kwadratów wprowadzonych liczb to " + sum);
+    }
```

Comment:
Zdefiniuj metody w taki sposób, aby były niezależne od siebie. Powinny przyjmować logiczne parametry i zwracać wynik. Niezależnie od tego w jakiej kolejności metody będą wywoływane, to nie muszą być uzależnione od innych